heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,19 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean confirmWithUser(int oldContainerCount, int newContainerCount) {
+    LOG.info(String.format("The present aurora job has %d containers. "
+        + "After update there will be %d containers. "
+        + "Please make sure there are sufficient resources to update this job. "
+        + "Continue update? [y/N]: ", oldContainerCount, newContainerCount));
+    Scanner scanner = new Scanner(System.in);
```

Comment:
Confirmation with the user should not be for all cases. If the user knows the behavior - it is annoying everytime to say yes/no - furthermore think about the fact that heron update is running in CI/CD - yes/no will not work very well.

Reply:
This verification can be done at the cli level itself - rather than doing it in the core.

Reply:
furthermore, there is no way for the user to verify whether the resources are available. Resource checking should be automatic.

Reply:
There is a toggle switch for this prompt. The default value is off, no change to the existing behavior.